src/index.js

```diff
@@ -0,0 +1,245 @@
+/* global document */
+/* eslint-disable eqeqeq */
+import projectObject from './project';
+import todoObject from './todo';
+import {
+  mainContent,
+  selectProject,
+  addProjectBtn,
+  toDoBtn,
+  projectForm,// eslint-disable-line 
+  projectName,
+  createProject,
+  cancelProject,
+  myTodoForm, // eslint-disable-line
+  todoTitle,
+  todoDescription,
+  todoDueDate,
+  todoPriority,
+  editTodo,
+  submitTodo,
+  cancelTodo,
+  todoLists,
+  todoListDetails,
+  domObjects,
+} from './domElements';
+// import './styles.css';
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history.